db/migrate/20170109045732_add_scores_to_sheets.rb

```diff
@@ -0,0 +1,6 @@
+class AddScoresToSheets < ActiveRecord::Migration
+  def change
+    add_column :sheets, :home_team_score_row, :integer, :array => true
+    add_column :sheets, :away_team_score_row, :integer, :array => true
```

Comment:
Nope! At least postgres supports arrays by default. So, for example, you can do:

`Sheet.new(:home_team_score_row => [1,2,3,3,4,5])`